syntax = "proto3";

package calculator;

service Calculator {
  rpc Add (AddRequest) returns (AddResponse);
}

message AddRequest {
  int32 operand1 = 1;
  int32 operand2 = 2;
}

message AddResponse {
  int32 result = 1;
}
